src/panels/config/entities/ha-config-entities.ts

```diff
@@ -527,21 +536,34 @@ export class HaConfigEntities extends LitElement {
                 <div class="header-btns">
                   ${!this.narrow
                     ? html`
-                        <mwc-button @click=${this._enableSelected}
-                          >${this.hass.localize(
-                            "ui.panel.config.entities.picker.enable_selected.button"
-                          )}</mwc-button
-                        >
-                        <mwc-button @click=${this._disableSelected}
-                          >${this.hass.localize(
-                            "ui.panel.config.entities.picker.disable_selected.button"
-                          )}</mwc-button
-                        >
-                        <mwc-button @click=${this._hideSelected}
-                          >${this.hass.localize(
-                            "ui.panel.config.entities.picker.hide_selected.button"
-                          )}</mwc-button
-                        >
+                        ${this._selectedCanEnable
```

Comment:
Instead of not shown, these should be disabled

Reply:
Done for buttons. 

Can I still hide the icons? I don't really like disabled icons for the UI, since they are pretty much the same color, and they lose the tooltip.